framework/doc/content/source/auxkernels/VariableGradientComponent.md

```diff
@@ -1,31 +1,8 @@
 # VariableGradientComponent
 
-!alert! construction title=Undocumented Class
-The VariableGradientComponent has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
-```markdown
-# VariableGradientComponent
-
-!syntax description /AuxKernels/VariableGradientComponent
-
-## Overview
-
-!! Replace these lines with information regarding the VariableGradientComponent object.
```

Comment:
Can you please add a few sentences about the object and how it can be used.

Reply:
ok I tried to find something to say, it's a pretty straight forward object!